src/android/com/microsoftopentechnologies/intellij/wizards/createvm/CreateVMWizard.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2014 Microsoft Open Technologies Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.microsoftopentechnologies.intellij.wizards.createvm;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.ui.wizard.WizardDialog;
+import com.microsoftopentechnologies.intellij.serviceexplorer.azure.vm.VMServiceModule;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class CreateVMWizard extends WizardDialog<CreateVMWizardModel> {
+    public CreateVMWizard(VMServiceModule node) {
+        super(true, true, new CreateVMWizardModel(node));
+    }
+
+    @Override
+    protected Dimension getWindowPreferredSize() {
+        this.getWindow();
+        this.setResizable(false);
+        return new Dimension(790, 467);
+    }
+
+    @Override
+    protected JComponent createSouthPanel() {
+        JComponent southPanelComp = super.createSouthPanel();
+
+        if (southPanelComp instanceof JPanel) {
```

Comment:
This is done in order to hide the Help button from the wizard, and is tightly coupled to the current implementation provided by IntelliJ; this is the same code already present in the other wizard.
The type verification is being done just to perform a safe cast, and we are traversing the components according to the known structure. If any of this validations should fail, that would mean that the known implementation had changed, and in that case the Help button would be shown.
Unfortunately there is no extension point to hide or disable the Help button.